cypress/integration/searchPerson.test.ts

```diff
@@ -54,6 +54,32 @@ describe('Search People', () => {
       .get('.alert-warning', { timeout: 60000 })
       .should('contain', 'No people found.')
   })
+
+  it('search with https ORCHID', () => {
```

Comment:
It's "ORCID" not "ORCHID", it's not named after the plant, an acronym for "Open Researcher and Contributor ID"